server/tinylicious/package-lock.json

```diff
@@ -1718,6 +1718,15 @@
         "is-glob": "~4.0.1",
         "normalize-path": "~3.0.0",
         "readdirp": "~3.5.0"
+      },
```

Comment:
Let's also take out the changes in this package-lock so that the changes in this PR are contained to the single tinylicious-client package that's being modified 😄